<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CondaInfoDialog</class>
 <widget class="QDialog" name="CondaInfoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>746</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Conda Information</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="iconLabel">
       <property name="text">
        <string notr="true">icon</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&lt;h2&gt;Conda Information&lt;/h2&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>conda Version:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="condaVersionLabel"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>conda-build Version:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="condaBuildVersionLabel"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>python Version:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="pythonVersionLabel"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="activeEnvironmentLabel">
       <property name="text">
        <string>Active Environment:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="activeEnvironmentEdit"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>User Configuration:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="userConfigEdit"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>System Configuration:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="systemConfigEdit"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="configurationsLabel">
       <property name="text">
        <string>Populated Configurations:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QPlainTextEdit" name="configurationsEdit"/>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Base Environment:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="baseEnvironmentEdit"/>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Channel URLs:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QPlainTextEdit" name="channelsEdit"/>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Package Cache:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QPlainTextEdit" name="cachesEdit"/>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Environment Directories:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QPlainTextEdit" name="envDirsEdit"/>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Platform:</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QLabel" name="platformLabel"/>
     </item>
     <item row="13" column="0">
      <widget class="QLabel" name="useragentLabel">
       <property name="text">
        <string>User-Agent:</string>
       </property>
      </widget>
     </item>
     <item row="13" column="1">
      <widget class="QLineEdit" name="useragentEdit"/>
     </item>
     <item row="14" column="0">
      <widget class="QLabel" name="uidGidLabel">
       <property name="text">
        <string>UID:GID:</string>
       </property>
      </widget>
     </item>
     <item row="14" column="1">
      <widget class="QLabel" name="uidGidDataLabel"/>
     </item>
     <item row="15" column="0">
      <widget class="QLabel" name="netrcLabel">
       <property name="text">
        <string>netrc File:</string>
       </property>
      </widget>
     </item>
     <item row="15" column="1">
      <widget class="QLineEdit" name="netrcEdit"/>
     </item>
     <item row="16" column="0">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>Offline Mode:</string>
       </property>
      </widget>
     </item>
     <item row="16" column="1">
      <widget class="QCheckBox" name="offlineCheckBox">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string>conda-env Version:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="condaEnvVersionLabel"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CondaInfoDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CondaInfoDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
